bindings/kepler.gl-jupyter/js/lib/keplergl/components/app.js

```diff
@@ -100,11 +100,11 @@ function App() {
           <title>Kepler.gl Jupyter</title>
           <link
             rel="stylesheet"
-            href="http://d1a3f4spazzrp4.cloudfront.net/kepler.gl/uber-fonts/4.0.0/superfine.css"
+            href="//d1a3f4spazzrp4.cloudfront.net/kepler.gl/uber-fonts/4.0.0/superfine.css"
```

Comment:
Maybe just change this to `https://` instead?

Reply:
What is holding up this PR? I have an https jupyterhub environment that would benefit from this fix.